src/plugins/ep_lightship/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "ep_lightship",
+  "version": "0.1.0",
+  "description": "Etherpad light plugin that integrates lightship health checks",
```

Comment:
should not be `Etherpad lite` ? I'm lost in this project between `lite` and `light`

Reply:
This is a typo. You are right: `lite` is the right way to call it. :wink: